src/components/Faq/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {FaqStyleTitle, FaqStyleContent} from './styles'
+
+class FAQ extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      clickedOn: false
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+  
+  handleClick() {
+    this.setState({
+      clickedOn: !this.state.clickedOn
+    });
+  }
+  render() {
+    const {title, answer} = this.props;
+    const {clickedOn} = this.state
+    let showAnswer;
+    if (clickedOn) {
+      showAnswer = <div> {answer.answer}</div>
+    }
+    return ( 
+      <div> 
+        <FaqStyleTitle onClick={this.handleClick}> 
+          {title} 
+        </FaqStyleTitle> 
+        <FaqStyleContent> 
+          { clickedOn && <div>{showAnswer}</div> } 
```

Comment:
Just put answer.answer in here, no need for the if statement on line 22. You can remove lines 22-24 completely.